pkg/util/validation.go

```diff
@@ -18,19 +18,32 @@ package util
 
 import (
 	"regexp"
-	"strings"
 )
 
-// IsDNSLabel tests for a string that conforms to the definition of a label in
-// DNS (RFC 1123).
-func IsDNSLabel(value string) bool {
-	return IsDNS1123Label(value)
+const LabelValueFmt string = "[-A-Za-z0-9_.]*"
```

Comment:
I don't like the idea of explicitly connecting qualified names with label values (the purist in me is just unable to write the thing you suggested). I'm OK writing it as exactly the same regexp, as long as it's possible to change either one without changing the other.